packages/compass-components/src/components/cancel-loader.tsx

```diff
@@ -1,5 +1,4 @@
-/** @jsx jsx */
-import { css, jsx } from '@emotion/react';
+import { css } from '@emotion/css';
```

Comment:
Moved to using this emotion/css syntax that uses `className` instead of the emotion/react `css` prop on components. This is because it was causing an error when run in electron's renderer process: `RangeError: WebAssembly.Compile is disallowed on the main thread, if the buffer size is larger than 4KB. Use WebAssembly.compile, or compile on a worker thread`. I think it was trying some server side rendering magic that uses web assembly and that would cause the error since it was generating something too big. Might be fixable, but I think this will be a fine workaround, will update https://jira.mongodb.org/browse/COMPASS-5219 after this is merged, since it's probably gone away unless we want to change back to the emotion/react later.

Reply:
Yikes, good catch! We should get to the bottom of this as debugging these styles is very hard now that the component names seem to be gone from styles now

Reply:
Updated the ticket to reflect these changes 👍